pyserini/vectorizer.py

```diff
@@ -0,0 +1,64 @@
+import math
+import numpy as np
+from typing import List
+from scipy.sparse import csr_matrix
```

Comment:
Do we need to update [requirements.txt](https://github.com/castorini/pyserini/blob/master/requirements.txt) to include `numpy` and `scipy`?

Reply:
yes

Reply:
added